utils/lang_checker.py

```diff
@@ -0,0 +1,66 @@
+import glob
+import argparse
+
+# Determines unused lang keys
+# Run by going to the root dir and using `python ./utils/lang_checker.py`
+# It will print out all unused lang keys
+# Can also automatically remove unused lang keys
+
+parser = argparse.ArgumentParser("lang_checker")
+parser.add_argument("--lang", help="The lang file to check", 
+                    type=str, default='./common/locales/en-US/main.ftl', required=False)
```

Comment:
could we update the script to also remove unused keys on the other locales? example spanish, portuguese etc

Reply:
you can already define which lang file to use. unless you mean it should check all files by default?